src/schemas/execute.yaml

```diff
@@ -1,7 +1,5 @@
 EthSimulatePayload:
   title: Arguments for multi call
-  required:
-    - blockStateCalls
```

Comment:
I am pretty sure this is not optional.  An `eth_simulateV1` without this doesn't make any sense.